src/main/webapp/js/userMapData.json

```diff
@@ -95,7 +103,8 @@
       "Bahrain Polytechnic"
     ],
     "Botswana": [
-      "Alpha"
+      "Alpha",
+      "University of Botswana"
     ],
     "Brazil": [
       "AEDB",
```

Comment:
What I meant is `AEDB` is probably same as `Associação Educacional Dom Bosco`. Not your fault, but you can fix it by deleting `AEDB`.